great_expectations/rule_based_profiler/parameter_builder/metric_multi_batch_parameter_builder.py

```diff
@@ -146,7 +126,7 @@ def _build_parameters(
             parameters=parameters,
         )
 
-        if reduce_scalar_metric and metric_values.shape[0] == 1:
```

Comment:
Why is this changing? Could we add a comment here?

Reply:
@Shinnnyshinshin Previous was a bug (adding comment -- in next commit).